Makefile

```diff
@@ -265,3 +270,13 @@ install: build
 	scripts/dev_install.sh -p $(GOPATH1)/bin
 
 .PHONY: default fmt vet lint check_license check_shell sanity cover prof deps build test fulltest shorttest clean cleango deploy node_exporter install %gen gen NONGO_BIN
+
+### TARGETS FOR CICD PROCESS
+
+ci-deps:
+	scripts/configure_dev-deps.sh && \
+	scripts/check_deps.sh
+
+ci-build: buildsrc gen
+	mkdir -p $(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH) && \
+	PKG_ROOT=$(SRCPATH)/tmp/node_pkgs/$(OS_TYPE)/$(ARCH) NO_BUILD=True VARIATIONS=literally_anything scripts/build_packages.sh $(OS_TYPE)/$(ARCH)
```

Comment:
I'm abit confused about the `VARIATIONS=literally_anything`.. could you provide some more context ?